extension/first-run.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8"/>
+    <title>Firefox Relay: By invitation only</title>
+    <link rel="stylesheet" href="/css/relay.css" />
+    <link rel="stylesheet" href = "/css/first-run.css" />
+    <script src="/js/first-run.js"></script>
+  </head>
+  <body class="first-run">
+    <h1 class="center first-run-h1 ff-Met">Firefox Relay: By invitation only</h1>
+    <main class="first-run-main">
+      <img class="first-run-img" src="/images/first-run-export.svg" alt="" aria-hidden="true">
+      <div class="first-run-content">
+        <p class="first-run-p">You’ve successfully installed the Firefox Relay add-on. To get started, sign in to your Firefox account to validate your invitation.</p>
+        <button id="first-run-sign-in" class="first-run-sign-in">Sign In</button>
+        <p class="first-run-join-waitlist">(Don’t have an invite?<a target="_blank" id="first-run-join-waitlist-link" class="first-run-join-waitlist-link" href="">Join the waitlist</a>. The next round of invitations go out soon.)</p>
```

Comment:
Hey, so dor whatever reason, the href in this file was not being updated during the build so I added an event listener in `first-run.js` that listens for these link clicks, appends a bunch of UTMs, then opens the homepage. We could send folks directly into the FxA flow but if they're joining the waitlist, it seems like it might make more sense to send them first to the homepage for more context.

Also, the CTAs no longer change value based on whether the addon is installed so anyone with the addon will still see the "Join the Waitlist" buttons.

Reply:
My eyes think that there should be a space after "invite?<a target="..."`> otherwise the "Join the waitlist" linked text might be too close (unless you're fiddling w/ spacing using CSS).

Reply:
I'm also not a UX/copy person, but three sentences of text in parenthesis seems like a bit much parenthetical text.

Reply:
Ah, I see it there now. That makes sense. @pdehaan - I think we're still expecting some more content tweaks by Monday.